### PR TITLE
Add admin API fixture for Shipping Profile

### PR DESCRIPTION
**Pull request by l4u**
#476

### CODE REVIEW

File: integration-tests/docs/test-input/admin/shipping-profile.js [hunk 1]
```diff
@@ -1 +1,29 @@
...
+module.exports = {
+  operationId: "GetShippingProfilesShippingProfile",
+  buildEndpoint: (id) => `/admin/shipping-profiles/${id}`,
+  setup: async (manager) => {
+    const newProfile = await manager.create(ShippingProfile, {
+      name: "shipping_profile_name",
+      type: "default"
+    })
+    await manager.save(newProfile)
+    console.log(newProfile)
```
Comment (olivermrbl) [inline integration-tests/docs/test-input/admin/shipping-profile.js:17]: Remove

**Review by olivermrbl — CHANGES_REQUESTED** [review]
All good! Just a single `console.log` we need to get rid off :)

**Comment by l4u** [comment]
removed the extra `console.log`

**Review by olivermrbl — APPROVED** [review]
Many thanks for the contribution ❤️

**Pull request merged**